workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/HyscaleDeployServiceCommand.java

```diff
@@ -118,8 +116,8 @@
     private DeployComponentInvoker deployComponentInvoker;
     
     @Autowired
-    private ServiceSpecMapper serviceSpecMapper;
-
+    private WorkflowUtil workflowUtil;
```

Comment:
The duplicated code that has been moved to workflow util just deals with some delta processing on service spec & profile ? Why should we name it Workflow Util ?